.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Bug report
+about: Report a problem with odo to help us resolve it and improve
+---
+[kind/bug]
+
+<!--
+
+Welcome! - We kindly ask that you:
+
+  1. Fill out the issue template below - not doing so needs a good reason.
+  2. Use the chat and talk to us if you have a question rather than a bug or feature request.
+
+The chat room is at: https://chat.openshift.io/developers/channels/odo
+
+NOTE: Not filling out the issue template needs a good reason, as otherwise it
+may take a lot longer to find the problem, not to mention it can take up a lot
+more time which can otherwise be spent on development. Please also take the
+time to help us debug the issue by collecting relevant information, even if
+it doesn't seem to be relevant to you. Thanks!
+
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+What versions of software are you using?
+----------------
+Operating System :
+
+odo version      : 
+<!--    you can find the version by running `odo version`
+-->
+
```

Comment:
Openshift version?
May help if trying some features available only since specific versions of openshift...(The command itself may say that but might help to have it)?

Reply:
if I'm not mistaken `odo version` also reports oc  and openshift version

Reply:
Ack :)
But the statement seems to suggest to me that user has to give me operating system, and odo version(which is obtained from command `odo version`) instead of output of `odo version` which gives me openshift version :)
Otherwise looks good to me :)

Reply:
ack :) made it more clear